Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -407,6 +407,30 @@ extension KeyboardShortcuts {
 	- Note: This method is not affected by `.removeAllHandlers()`.
 	*/
 	@available(macOS 10.15, *)
+	public static func events(for name: Name) -> AsyncStream<KeyboardShortcuts.EventType> {
+		AsyncStream { continuation in
+			let id = UUID()
+
+			streamKeyDownHandlers[name, default: [:]][id] = {
+				continuation.yield(.keyDown)
+			}
+			streamKeyUpHandlers[name, default: [:]][id] = {
+				continuation.yield(.keyUp)
+			}
+
+			registerShortcutIfNeeded(for: name)
+
+			continuation.onTermination = { _ in
+				streamKeyDownHandlers[name]?[id] = nil
+				streamKeyUpHandlers[name]?[id] = nil
+
+				unregisterShortcutIfNeeded(for: name)
+			}
+		}
+	}
+
+	@available(macOS 10.15, *)
+	@available(*, deprecated, message: "Use events(for:) instead")
 	public static func on(_ type: EventType, for name: Name) -> AsyncStream<Void> {
 		AsyncStream { continuation in
```

Comment:
Can you make the `on` method use `events()` internally? To reduce the amount of duplicated code.